bcp/client/src/webpack/components/TableSort.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import SortIcon from '../../../../public/icons/descending.svg?component';
```

Comment:
Unless you plan to reference these .svg files externally they shouldn't need to be in public. Webpack will take care of bundling any svgs you are using as components. I feel they should be somewhere under client/src.